.github/workflows/conda.yml

```diff
@@ -26,22 +26,28 @@ jobs:
     name: Build (and upload)
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3.3.0
+      - name: Checkout source
+        uses: actions/checkout@v3
         with:
-          fetch-depth: 0
+          # Needed by versioneer.
+          # Must be higher than the number of commits since the latest release.
+          fetch-depth: 200
```

Comment:
I don't know what happens when a PR was first opened months ago, so it forked from main more than 200 commits ago. If this causes problems we can increase this number or just squash the PR.